src/integration.ts

```diff
@@ -143,6 +143,25 @@ export class ProfilingIntegration implements Integration {
           PROFILE_QUEUE.splice(profileIndex, 1);
           const profile = createProfilingEvent(cpuProfile, profiledTransaction);
 
+          if (client.emit && profile) {
+            const integrations =
+              // @ts-expect-error _integrations is private
+              typeof client._integrations === 'object' &&
```

Comment:
Just a tip: you can use `client['_integrations']` to access private fields without TS complaining.